config/locales/es.yml

```diff
@@ -1970,12 +1964,12 @@ es:
 
             <p class="text--small spacing-above-15">Su nivel de ingresos no debe incluir pagos de estímulo, prestaciones públicas o Seguridad de Ingreso Suplementario (SSI)</p>
           options:
-            12500_to_25000: De $15,000 a 30,000 dólares
-            1_to_12500: De $1 a $15,000 dólares
-            25000_to_40000: De $30,000 a 45,000 dólares
-            40000_to_66000: De $45,000 a 67,000 dólares
-            66000_to_79000: De $67,000 a 84,000 dólares
-            over_79000: más de $84,000
+            12500_to_25000: "$15,000 - 30,000"
```

Comment:
This is used on the triage income level page in the GYR flow, so I think it should remain spanish no? Also asked about this last time [here](https://cfastaff.slack.com/archives/C087E6C7NBZ/p1738097450028099). Seemed like it was going to get changed, could be worth double checking